### PR TITLE
Validate form

### PR DESCRIPTION
**Pull request by edgardsoza**
This project was created to implement simple validation. Even there are just commits from mine, Steven and I worked together on this one. 

The content of the email field has to be in lower case. !!Done

You should implement the following interactions:

When the user submits the form, you check if the email is in lower case. Made by me. 
If the validation is OK, the form is sent. Made by Steven.
If the validation is not OK, you show an error message to the user near the submit button informing them of the error and the form is not sent. Made by me.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,7 +9,14 @@
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Inter&display=swap" rel="stylesheet">
+<<<<<<< Updated upstream
```
Comment (rustam-tolipov) [inline index.html]: [OPTIONAL] Kindly remove this line. Please, accept my suggestion.
```suggestion
```

File: index.html [hunk 2]
```diff
@@ -9,7 +9,14 @@
...
     <script src="https://kit.fontawesome.com/b33cf67da8.js" crossorigin="anonymous"></script>    
+=======
```
Comment (rustam-tolipov) [inline index.html]: [OPTIONAL] Kindly remove this line. Please, accept my suggestion.
```suggestion
```

File: index.html [hunk 3]
```diff
@@ -9,7 +9,14 @@
...
+    <script src="https://kit.fontawesome.com/b33cf67da8.js" crossorigin="anonymous"></script>
+    <script src="./popup-portfolio.js" defer></script>
+    <script src="./validation.js" defer></script>
+    <link rel="stylesheet" type="text/css" href="popup-portfolio.css">
+>>>>>>> Stashed changes
```
Comment (rustam-tolipov) [inline index.html]: [OPTIONAL] Kindly remove this line. Please, accept my suggestion.
```suggestion
```

**Review by rustam-tolipov — APPROVED** [review]
Hi @edgardsoza, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Developer-512px-10

## Highlights
1. Perfect validation 👍
2. Stying of validation is above! 🫠

## Optional suggestions
_Kindly check the comments below_

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (edgardsoza) [reply index.html]: I fixed it already

**Pull request merged**